blocks/address/src/app.tsx

```diff
@@ -0,0 +1,677 @@
+                        lineHeight: "18px",
+                        color: ({ palette }) => palette.common.black,
+                        marginBottom: 0.5,
+                      }}
+                    >
+                      {label}
+                    </Typography>
+
+                    <Typography
+                      variant="microText"
+                      sx={{
+                        fontSize: 13,
+                        lineHeight: "18px",
+                        color: ({ palette }) => palette.gray[50],
+                      }}
+                    >
+                      {option.country}
+                    </Typography>
+                  </Stack>
+                );
+              }}
+              PaperComponent={({ children, ...props }) => (
+                <Paper
+                  {...props}
+                  sx={{
+                    filter:
+                      "drop-shadow(0px 11px 30px rgba(61, 78, 133, 0.04)) drop-shadow(0px 7.12963px 18.37px rgba(61, 78, 133, 0.05)) drop-shadow(0px 4.23704px 8.1px rgba(61, 78, 133, 0.06)) drop-shadow(0px 0.203704px 0.62963px rgba(61, 78, 133, 0.07))",
+                    border: ({ palette }) => `1px solid ${palette.gray[20]}`,
+                    boxShadow: "none",
+                    [`.${autocompleteClasses.listbox}`]: {
+                      padding: "0px",
+                      maxHeight: "unset",
+                      [`.${autocompleteClasses.option}`]: {
+                        alignItems: "flex-start",
+                        paddingX: ({ spacing }) => `${spacing(2.5)} !important`,
+                        paddingY: 1.25,
+                      },
+                    },
+                  }}
+                >
+                  {children}
+                </Paper>
+              )}
+              sx={{
+                [`.${autocompleteClasses.input}`]: {
+                  paddingLeft: "0 !important",
+                },
+                [`.${autocompleteClasses.inputRoot}`]: {
+                  paddingX: ({ spacing }) => `${spacing(2.75)} !important`,
+                },
+              }}
+            />
+
+            {suggestionsError ? (
+              <Box
+                display="flex"
+                flexDirection="column"
+                justifyContent="center"
+                gap={1}
+              >
+                <Typography
+                  variant="smallTextLabels"
+                  sx={{
+                    fontWeight: 700,
+                    fontSize: 13,
+                    lineHeight: 1,
+                    letterSpacing: "-0.02em",
+                    color: ({ palette }) => palette.black,
+                  }}
+                >
+                  <Box
+                    component="span"
+                    sx={{ color: ({ palette }) => palette.red[60] }}
+                  >
+                    Error connecting
+                  </Box>{" "}
+                  to the Block Protocol
```

Comment:
this may be implemented in other ways than connecting to blockprotocol.org
```suggestion
                  to the Mapbox API
```